pkg/agave/validator.go

```diff
@@ -38,6 +52,10 @@ func (cmd *InstallCommand) Env() map[string]string {
 		env["VALIDATOR_VERSION"] = *cmd.Version
 	}
 
+	if cmd.Metrics != nil && cmd.Metrics.SolanaMetricsURL != "" {
+		env["SOLANA_METRICS_CONFIG"] = cmd.Metrics.SolanaMetricsURL
+	}
+
```

Comment:
```suggestion
	if cmd.Metrics != nil {
           metrics, env := cmd.Metrics.ToEnv()
           	if err != nil {
		return env
	}
        env["SOLANA_METRICS_CONFIG"] = metrics
	}
```

Can add conversion method to metrics struct which returns the env string result or an error.

A note to refactor to allow errors to bubble up instead of no oping. The type system enforces the existence of URL on metrics but we should validate the url is valid format.